package.json

```diff
@@ -209,92 +209,111 @@
+    "react-router-redux": "5.0.0-alpha.9",
+    "redux": "4.0.0",
     "redux-immutable": "4.0.0",
-    "redux-saga": "0.15.3",
+    "redux-saga": "0.16.0",
     "reselect": "3.0.1",
     "sanitize.css": "4.1.0",
-    "styled-components": "2.0.0",
+    "styled-components": "2.3.0",
     "warning": "3.0.0",
-    "whatwg-fetch": "2.0.3"
+    "whatwg-fetch": "2.0.4"
   },
   "devDependencies": {
-    "add-asset-html-webpack-plugin": "2.0.1",
+    "add-asset-html-webpack-plugin": "2.1.3",
     "babel-cli": "6.24.1",
     "babel-core": "6.24.1",
     "babel-eslint": "7.2.3",
-    "babel-loader": "7.1.0",
+    "babel-loader": "7.1.4",
     "babel-plugin-dynamic-import-node": "1.0.2",
     "babel-plugin-react-intl": "2.3.1",
     "babel-plugin-react-transform": "2.0.2",
     "babel-plugin-styled-components": "1.1.4",
     "babel-plugin-transform-es2015-modules-commonjs": "6.24.1",
     "babel-plugin-transform-react-constant-elements": "6.23.0",
     "babel-plugin-transform-react-inline-elements": "6.22.0",
+    "babel-plugin-transform-react-jsx-source": "6.22.0",
     "babel-plugin-transform-react-remove-prop-types": "0.4.5",
     "babel-preset-env": "1.5.1",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-0": "6.24.1",
     "circular-dependency-plugin": "3.0.0",
     "coveralls": "2.13.1",
-    "css-loader": "0.28.4",
-    "enzyme": "2.8.2",
+    "css-loader": "0.28.11",
+    "enzyme": "3.3.0",
+    "enzyme-adapter-react-16": "1.1.1",
+    "enzyme-to-json": "^3.3.4",
     "eslint": "3.19.0",
     "eslint-config-airbnb": "15.0.1",
     "eslint-config-airbnb-base": "11.2.0",
-    "eslint-import-resolver-webpack": "0.8.3",
+    "eslint-import-resolver-webpack": "0.8.4",
     "eslint-plugin-import": "2.7.0",
     "eslint-plugin-jsx-a11y": "5.0.3",
     "eslint-plugin-react": "7.0.1",
     "eslint-plugin-redux-saga": "0.3.0",
     "eventsource-polyfill": "0.9.6",
-    "exports-loader": "0.6.4",
-    "file-loader": "0.11.1",
-    "html-loader": "0.4.5",
-    "html-webpack-plugin": "2.29.0",
-    "image-webpack-loader": "2.0.0",
-    "imports-loader": "0.7.1",
+    "exports-loader": "0.7.0",
+    "file-loader": "1.1.11",
+    "html-loader": "0.5.5",
+    "html-webpack-plugin": "3.0.6",
+    "husky": "0.14.3",
+    "image-webpack-loader": "4.2.0",
+    "imports-loader": "0.8.0",
     "jest-cli": "20.0.4",
+    "jest-styled-components": "^5.0.1",
     "lint-staged": "3.5.1",
     "ngrok": "2.2.9",
     "node-plop": "0.7.0",
     "null-loader": "0.1.1",
-    "offline-plugin": "4.8.1",
+    "offline-plugin": "4.9.0",
     "plop": "1.8.0",
     "pre-commit": "1.2.2",
-    "react-test-renderer": "15.6.1",
-    "rimraf": "2.6.1",
+    "react-test-renderer": "16.2.0",
+    "rimraf": "2.6.2",
     "shelljs": "0.7.7",
-    "style-loader": "0.18.1",
-    "url-loader": "0.5.8",
-    "webpack": "3.5.5",
-    "webpack-dev-middleware": "1.11.0",
-    "webpack-hot-middleware": "2.18.0"
+    "style-loader": "0.20.3",
+    "stylelint": "8.3.1",
```

Comment:
You added the `stylelint` packages, but no configuration for them.

Reply:
Here's the one I use if it helps:
```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "selector-pseudo-class-no-unknown": [
      true,
      {
        "ignorePseudoClasses": [
          "global"
        ]
      }
    ],
    "no-descending-specificity": null,
    "declaration-colon-newline-after": null,
    "value-list-comma-newline-after": null,
  }
}
```